### PR TITLE
ENH: Add noise power spectrum to foms

### PR DESCRIPTION
**Pull request by adler-j**
This adds NPS computations, I guess we should also add MTF.

### CODE REVIEW

**Comment by adler-j** [comment]
Could use a review

File: odl/contrib/fom/supervised.py [hunk 1]
```diff
@@ -628,6 +629,46 @@ def psnr(data, ground_truth, normalize=False):
     else:
         return 20 * np.log10(max_true) - 10 * np.log10(mse_result)
 
+
+def noise_power_spectrum(data, ground_truth, radial=False):
+    """Return the Noise Power Spectrum (NPS).
+
+    The NPS is given by the square of the fourier transform of the noise.
```
Comment (kohr-h) [inline odl/contrib/fom/supervised.py]: squared magnitude
Fourier

File: odl/contrib/fom/supervised.py [hunk 2]
```diff
@@ -628,6 +629,46 @@ def psnr(data, ground_truth, normalize=False):
...
+
+    Parameters
+    ----------
+    data : `DiscreteLp`-element
```
Comment (kohr-h) [inline odl/contrib/fom/supervised.py]: no dash

File: odl/contrib/fom/supervised.py [hunk 3]
```diff
@@ -628,6 +629,46 @@ def psnr(data, ground_truth, normalize=False):
...
+        Input data or reconstruction.
+    ground_truth : `DiscreteLp`-element
+        Reference to compare ``data`` to.
+    radial : bool
+        If true, compute the radial NPS.
+
+    Returns
+    -------
+    noise_power_spectrum : `DiscreteLp`-element
+        Noise power spectrum. The space is the fourier transform of
```
Comment (kohr-h) [inline odl/contrib/fom/supervised.py]: Just repeating the name?
Fourier
and "Fourier transform of the space" isn't really a thing, rather "Fourier space corresponding to ..." or "frequency space" or so.

File: odl/contrib/fom/supervised.py [hunk 4]
```diff
@@ -628,6 +629,46 @@ def psnr(data, ground_truth, normalize=False):
...
+        ``data.space``, and hence the axes indicate frequency.
+        If ``radial`` is True, this is an element in a one-dimensional space
```
Comment (kohr-h) [inline odl/contrib/fom/supervised.py]: \`\`True\`\`

**Comment by adler-j** [comment]
Merge after CI

**Pull request merged**